client/my-sites/migrate/section-migrate.jsx

```diff
@@ -472,6 +472,14 @@ class SectionMigrate extends Component {
 					{ this.renderStartTime() }
 					{ this.renderProgressBar() }
 					{ this.renderProgressList() }
+					<p class="migrate__note">
+						{ translate(
+							"You can safely navigate away from this page if you need to {{br/}} we'll send you a notification when it's done",
```

Comment:
I think we should try to omit the `{{br/}}` and limit the width of the text with CSS.

Reply:
I agree with you; we should try not to add HTML elements inside of the localization string.

The CSS breaks the line, and now it looks:

![Markup on 2021-04-29 at 13:17:54](https://user-images.githubusercontent.com/1241413/116543722-81e6c700-a8ee-11eb-945f-cc55515f13c5.png)